### DISCUSSION TITLE
A map seems to use classic sound effects even though it shouldn't

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#21783] [Q&A] [answered]
I noticed while playing that [this map](https://osu.ppy.sh/beatmapsets/57984#osu/174515) seems to use the classic sound effects, even though I was using a custom skin I made based on the "argon" one. In the video, I'm using the default argon skin to ensure that wasn't an issue with my edited skin. I don't think I'm missing anything that would cause that.

You can hear the spinner use different sounds than argon, yet the options regarding the beatmap are all turned off (i.e. skin, colours and video):

https://user-images.githubusercontent.com/43073074/209451556-32ed65ae-0dfc-41da-91b1-3f5d06c72ef9.mp4

**Comment by Theighlin** [comment, marked as answer]
Just to make sure, does it still happen with beatmap hitsounds turned off? (You have them turned on in the video)

**Reply by Gabixel** [reply]
My bad here. I forgot there was that option. By disabling it you hear your skin as intended.
